errtrace.go

```diff
@@ -123,6 +133,11 @@ func (e *errTrace) Format(s fmt.State, verb rune) {
 	fmt.Fprintf(s, fmt.FormatString(s, verb), e.err)
 }
 
+// LogValue implements the [slog.LogValuer] interface.
+func (e *errTrace) LogValue() slog.Value {
+	return slog.StringValue(FormatString(e))
```

Comment:
no changes needed, but wanted to consider whether this should return a string value, or a group with multiple fields for message, trace, etc.

I think a string makes sense to start, and allows callers to choose how they want the error encoded, e.g., in their equivalent of the `LogAttr` function.